charts/opentelemetry-operator/values.schema.json

```diff
@@ -512,9 +512,25 @@
                 "featureGates": {
                     "type": "string",
                     "default": "",
+                    "title": "The featureGates to enable",
+                    "examples": ""
+                },
+                "featureGatesMap": {
```

Comment:
i believe the default is true, but we should explicitly set `additionalProperties` for validation.

Reply:
Fixed.